docs/QUICK_START.md

```diff
@@ -0,0 +1,202 @@
+- These 2.4 GHz band modules were also designed specifically for mLRS and arrive pre-flashed with mLRS version 1.3.
+- The mR24-30-TX module also includes the HC04 module but you may notice some interference since Bluetooth also uses the 2.4 GHz band.
+- These modules are also especially easy to flash via DFU over their USB C connection.
+- 2.4 GHz is a wider band which might support more simultaneous users.
+- These modules support 19 Hz, 31 Hz, 50, and FLRC(111 Hz) modes.
+- The Tx module kit does not include an antenna or JR bay case which must be 3D printed.  If you don't have access to a 3D printer, you can purchase a print from a number of 3D printing services.
+- If you don't need quite as much range, you can opt for one of the smaller and lighter [Selected ELRS 2.4 GHz receivers](ELRS_RECEIVERS.md#selected-elrs-receivers) such as the RadioMaster RP4TD.
+- Some AIO flight controller boards include a low power 2.4 GHz ELRS serial receiver which can also be used with this Tx module.
+
+### FrSky [R9M Tx module](FRSKY_R9.md#r9m-tx-module) with the [Bayck 915M Nano Pro](ELRS_RECEIVERS.md#selected-elrs-receivers) 900 MHz ELRS receiver
+- This option is not highly recommended for new users as it is more complicated and slightly less flexible, but is included here since it allows use of ELRS receivers you may already have.
+- This Tx module uses the SX1276 LoRa chip and supports only 19 Hz mode.  It works with many 900 MHz ELRS receivers.
+- The FrSky R9M can be flashed using your OpenTX or EdgeTX receiver.
+- No 3D printer is required, but transmit Power should be limited to less than 500 mW unless you add a fan.
+- The Bayck 915M Nano Pro ELRS receiver can be [flashed](ELRS_RECEIVERS.md#flashing) with mLRS firmware without removing it from your vehicle by using ArduPilot serial passthrough.
+- If you already have a R9MX or R9MM receiver, they are also compatible, but have lower power and are more difficult to [flash](FRSKY_R9.md#flash-r9-receivers-with-elrs-bootloader).
+- Unfortunately, the R9M has inverted serial port signals; if you choose this option you will probably also want to purchase an [ESP32 module](FRSKY_R9.md#esp32-wireless-bridge) such as the [M5 Stamp Pico Mate](https://shop.m5stack.com/products/m5stamp-pico-diy-kit) to use as a wireless bridge and handle the inverted serial port.
+
+## 2. Assemble your Tx module if required and install antennas on both the receiver and Tx module
+
+If you are using one of the MatekSys Tx module kits, print or purchase a JR bay case from the STL download available from [MatekSys](https://www.mateksys.com/?portfolio=mr900-30-tx#tab-id-1).
+
+If you will use the MatekSys HC-04 module to connect your ground station via Bluetooth, set all three [dip switches](https://MATEKSYS.md#tx-module-hc-04-bluetooth-notes) to the on position befoer assembling your Tx module.
```

Comment:
HC-04 or HC04? pl decide and then stick to your decission :)

" ... before ..."